### DISCUSSION TITLE
使用FlinkCDC接入mysql多表到doris启动后新增表数据没有同步，求助！

### DISCUSSION THREAD

**Discussion by echobigdata** [apache/doris#24386] [A - General / Q&A]
doris:2.0.1
flink:1.16.2
flink-doris-connector-1.16-1.4.0.jar
flink-sql-connector-mysql-cdc-2.3.0.jar
任务启动后在mysql新加一个表，doris没有同步过来，然后停止任务从保存点重启后只把新加表结构同步过来了，表数据仍然没同步。
![微信图片_20230914180702](https://github.com/apache/doris/assets/97231854/21a5e159-8309-4b81-9354-a35b2d76d04e)

**Comment by yagagagaga** [comment]
Could you provide taskmanager's log ? And why are you use SNAPSHOT jar?
